pkg/pillar/cmd/volumemgr/handlevolume.go

```diff
@@ -328,22 +322,30 @@ func maybeSpaceAvailable(ctx *volumemgrContext) {
 }
 
 // updateVolumeStatusRefCount updates the refcount in volume status
-// Refcount in volume status is sum of refount in volume config and volume ref config
+// Refcount in volume status is the number of all received VolumeConfigs and VolumeRefConfigs
 func updateVolumeStatusRefCount(ctx *volumemgrContext, vs *types.VolumeStatus) {
 	log.Tracef("updateVolumeStatusRefCount(%s)", vs.Key())
-	var vcRefCount, vrcRefCount uint
+	var vcRefCount, vrcRefCount uint // initialize to 0
```

Comment:
why do you need two counter variables here?

Reply:
for clarity :)